scripts/ci/jobs/gke_nongroovy_e2e_tests.py

```diff
@@ -13,6 +13,9 @@
 # set required test parameters
 os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
 
+# don't use postgres
+os.environ["ROX_POSTGRES_DATASTORE"] = "false"
```

Comment:
Should we plan to have postgres nongroovy tests?

Reply:
Yes :) (to both)